src/components/App/App.js

```diff
@@ -21,6 +23,19 @@ class App extends Component {
 		if (!this.state.prediction) this.setState({prediction: {index, img}});
```

Comment:
This looks to be from a prior PR but is worth mentioning, since [`setState`](https://reactjs.org/docs/react-component.html#setstate) is asynchronous (the calls are batched) this isn't guaranteed to be the value we expect it to be

Using the callback form allows this to always work:

```
this.setState(prevState => {
  if (prevState.prediction) {
    return null // All good, don't need to set it. Returning null doesn't cause rerender
  }

  return {
    prediction: { index, img },
 }
})
```